accepted/00000-use-dnf-for-rpm-repos.md

```diff
@@ -0,0 +1,177 @@
+- Feature Name: Use DNF for managing the RPM content
+- Start Date: 04-04-2019
+- RFC PR: https://github.com/uyuni-project/uyuni-rfc/pull/4
+
+# Summary
+[summary]: #summary
+
+Uyuni contains components that support importing external repositories;
+collecting and retrieving information on packages, products, modules, and other content;
+and hosting repositories to be consumed by Spacewalk clients. For RPM repositories,
+it currently uses Zypper and libsolv (and previously used YUM), per
+[a previously implemented RFC](https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00047-yum-replacement.md).
+
+However, this makes it difficult to run Uyuni on Red Hat Enterprise Linux and CentOS servers
+without replacing base packages (violating subscription support terms in the RHEL case),
+and makes it even more challenging to properly support newer extensions to RPM repositories,
+such as RH/Fedora modules introduced in Fedora 30 and RHEL 8.
```

Comment:
Supporting and managing the new app streams from RH/Fedora apparently requires more changes on the codebase apart from the reposync plugin. So this is something we will be investigating outside of the context of the reposync plugins.

The downloading part and metadata reading is also feasible to do with our current zypper plugin.